guides/common/modules/proc_converting-hosts-with-convert2rhel.adoc

```diff
@@ -0,0 +1,22 @@
+[id="converting-hosts-with-convert2rhel_{context}"]
+
+= Converting Hosts with Convert2RHEL
+
+Use the following procedure to perform the conversion.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* *>* *All Hosts* and select the target hosts you want to convert.
+You can use the *Search* field to filter the host list.
+. From the *Select Action* list, select Schedule Remote Job.
+. In the *Job category* field, select `Convert to RHEL`.
+. In the *Job template* field, verify that the Job Template is set to `Convert to RHEL`.
+. On the *Job Invocation* page, define the main job settings.
+. The *Search Query* field displays the name of any search you have preformed, or the list of hosts you have selected.
```

Comment:
```suggestion
The *Search Query* field displays the name of any search you have preformed, or the list of hosts you have selected.
```

This is not an instruction: so maybe leave it out or remove the number.

Reply:
Fixed.